.github/workflows/verify-push.yaml

```diff
@@ -1,12 +1,12 @@
 # This workflow will perform a test whenever there
 # is some change in code done to ensure that the changes
 # are not buggy and we are getting the desired output.
-name: Test Push without image
-on: [ push, workflow_dispatch ]
+name: Test Build and Push
+on: [ push, workflow_dispatch, pull_request_target ]
```

Comment:
Yes, I think this is pretty new feature https://docs.github.com/en/actions/reference/events-that-trigger-workflows#pull_request_target
I think it would be useful for few actions like this, where we are not able to do integration tests on pull request.